manager/pom.xml

```diff
@@ -14,182 +14,17 @@
   <name>SmartEvents :: Manager</name>
 
   <dependencies>
-    <dependency>
-      <groupId>io.quarkus</groupId>
```

Comment:
same here? A `manager` packaging-pom with `core`, `v1`, `v2` and `runner` as submodules?